blackbox/docs/general/builtins/scalar.rst

```diff
@@ -706,9 +706,12 @@ If no ``format_string`` is given the default format will be used::
     +-----------------------------+
     SELECT 1 row in set (... sec)
 
+.. _scalar-timezone:
```

Comment:
Should the link for the `timezone` scalar be further down? This section here is the description of `timezone` in the context of `date_format`

Reply:
done